<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>toolbyn</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Image/rec1.bmp</normalon>
     <activeon>:/Image/rec2.bmp</activeon>
     <selectedon>:/Image/DRAGON_256_1.bmp</selectedon>
    </iconset>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Image/rec2.bmp</normalon>
    </iconset>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="tristate">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>102</width>
     <height>73</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rBtnMan">
      <property name="text">
       <string>M</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rBtnFm">
      <property name="text">
       <string>F</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>102</width>
     <height>73</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
